django/db/backends/mysql/operations.py

```diff
@@ -347,8 +347,12 @@ def explain_query_prefix(self, format=None, **options):
         if analyze and self.connection.features.supports_explain_analyze:
             # MariaDB uses ANALYZE instead of EXPLAIN ANALYZE.
             prefix = 'ANALYZE' if self.connection.mysql_is_mariadb else prefix + ' ANALYZE'
-        if format and not (analyze and not self.connection.mysql_is_mariadb):
-            # Only MariaDB supports the analyze option with formats.
+        if format and not (
+            analyze and
+            not self.connection.mysql_is_mariadb and
+            self.connection.mysql_version < (8, 0, 21)
+        ):
+            # Only MariaDB and MySQL 8.0.20+ supports the analyze option with formats.
```

Comment:
Patch version 20 or 21?
Also, as I've commented on this line I think the s from supports can be chopped.